08-validation.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "CakeMap"
+title: "Validation"
```

Comment:
Ah yes - schoolboy error on my part. Note the titles wash away when the book is built